docs/en/latest/plugins/limit-conn.md

```diff
@@ -26,42 +26,47 @@ description: This document contains information about the Apache APISIX limit-co
-| redis_cluster_ssl        | boolean |  False | false       |                                   | If set to `true`, then uses SSL to connect to redis-cluster. Used when the `policy` attribute is set to `redis-cluster`.                                                                                                                                                                                                                                                                            |
-| redis_cluster_ssl_verify | boolean | False | false       |                                   | If set to `true`, then verifies the validity of the server SSL certificate. Used when the `policy` attribute is set to `redis-cluster`.                                                                                                                                                                                                                                                             |
-
-## Enable Plugin
-
-You can enable the Plugin on a Route as shown below:
+| Name       | Type    | Required | Default     | Valid values      | Description     |
+|------------|---------|----------|-------------|-------------------|-----------------|
+| conn       | integer | True     |     | > 0   | The maximum number of concurrent requests allowed. Requests exceeding the configured limit and below `conn + burst` will be delayed.      |
+| burst      | integer | True     |     | >= 0        | The number of excessive concurrent requests allowed to be delayed per second. Requests exceeding the limit will be rejected immediately.       |
+| default_conn_delay       | number  | True     |     | > 0    | Processing latency allowed in seconds for concurrent requests exceeding `conn + burst`, which can be dynamically adjusted based on `only_use_default_delay` setting.           |
+| only_use_default_delay   | boolean | False    | false       |      | If false, delay requests proportionally based on how much they exceed the `conn` limit. The delay grows larger as congestion increases. For instance, with `conn` being `5`, `burst` being `3`, and `default_conn_delay` being `1`, 6 concurrent requests would result in a 1-second delay, 7 requests a 2-second delay, 8 requests a 3-second delay, and so on, until the total limit of `conn + burst` is reached, beyond which requests are rejected. If true, use `default_conn_delay` to delay all excessive requests within the `burst` range. Requests beyond `conn + burst` are rejected immediately. For instance, with `conn` being `5`, `burst` being `3`, and `default_conn_delay` being `1`, 6, 7, or 8 concurrent requests are all delayed by exactly 1 second each. |
```

Comment:
inconsistent
![image](https://github.com/user-attachments/assets/181ba527-68c1-4818-8229-ad056de0157e)

Reply:
One is required or not; the other one is default value.

Reply:
OK